virttest/utils_test/qemu/__init__.py

```diff
@@ -679,6 +679,8 @@ def check_memory(self, vm=None):
             vm = self.env.get_vm(self.params["main_vm"])
         vm.verify_alive()
         threshold = float(self.params.get("threshold", 0.10))
+        if "ppc" in self.params.get("vm_arch_name"):
+            threshold = float(self.params.get("threshold", 0.15))
```

Comment:
Please set it into the configuration file and leave the comment about why it needs a little bit more on ppc platform.